swagger/v3/api_spec.json

```diff
@@ -211,1181 +211,14 @@
             }
           }
         }
-      },
```

Comment:
I think because API v3 has not been built fully these specs are removed as the endpoints don't exist. We however need those here to generate the draft API spec we are sharing with providers. Do you mind adding those back please?